<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaz_robotClass</class>
 <widget class="QMainWindow" name="interfaz_robotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interfaz_robot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f5f5;   /* gris claro */
    font-family: &quot;Segoe UI&quot;;
    font-size: 11pt;
    color: #333333;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_camara">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: #ffffff;
    border: 2px solid #d0d0d0;
    border-radius: 8px;
    margin-top: 20px;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    color: #0078d7;
    font-size: 12pt;
}
</string>
      </property>
      <property name="title">
       <string>CÁMARA</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnInicio">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ffffff;       
    border: 1px solid #cccccc;       
    border-radius: 6px;              
    padding: 6px 12px;               
    color: #333333;                  
    font-weight: bold;
}

/* Al pasar el ratón por encima */
QPushButton:hover {
    background-color: #e6f0ff;       /* azul muy claro (efecto hover) */
    border: 1px solid #99c2ff;
}

/* Al hacer clic */
QPushButton:pressed {
    background-color: #cce0ff;       /* azul un poco más fuerte */
    border: 1px solid #66a3ff;
}

/* Si está deshabilitado */
QPushButton:disabled {
    background-color: #f0f0f0;
    color: #aaaaaa;
    border: 1px solid #dddddd;
}
</string>
         </property>
         <property name="text">
          <string>Iniciar vídeo</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblInicio">
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #eeeeee;
    border: 1px solid #cccccc;
    border-radius: 4px;
    color: #777777;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCapturar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ffffff;       
    border: 1px solid #cccccc;       
    border-radius: 6px;              
    padding: 6px 12px;               
    color: #333333;                  
    font-weight: bold;
}

/* Al pasar el ratón por encima */
QPushButton:hover {
    background-color: #e6f0ff;       /* azul muy claro (efecto hover) */
    border: 1px solid #99c2ff;
}

/* Al hacer clic */
QPushButton:pressed {
    background-color: #cce0ff;       /* azul un poco más fuerte */
    border: 1px solid #66a3ff;
}

/* Si está deshabilitado */
QPushButton:disabled {
    background-color: #f0f0f0;
    color: #aaaaaa;
    border: 1px solid #dddddd;
}
</string>
         </property>
         <property name="text">
          <string>Capturar imagen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCaptura">
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #eeeeee;
    border: 1px solid #cccccc;
    border-radius: 4px;
    color: #777777;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGuardar">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #ffffff;       
    border: 1px solid #cccccc;     
    border-radius: 6px;             
    padding: 6px 12px;               
    color: #333333;                  
    font-weight: bold;
}

/* Al pasar el ratón por encima */
QPushButton:hover {
    background-color: #e6f0ff;       /* azul muy claro (efecto hover) */
    border: 1px solid #99c2ff;
}

/* Al hacer clic */
QPushButton:pressed {
    background-color: #cce0ff;       /* azul un poco más fuerte */
    border: 1px solid #66a3ff;
}

/* Si está deshabilitado */
QPushButton:disabled {
    background-color: #f0f0f0;
    color: #aaaaaa;
    border: 1px solid #dddddd;
}
</string>
         </property>
         <property name="text">
          <string>Guardar imagen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_robot">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    background-color: #ffffff;       /* blanco dentro de cada sección */
    border: 2px solid #d0d0d0;
    border-radius: 8px;
    margin-top: 20px;                /* espacio para el título */
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 10px;
    color: #0078d7;                  /* azul para títulos */
    font-size: 12pt;
}
</string>
      </property>
      <property name="title">
       <string>ROBOT</string>
      </property>
      <widget class="QGroupBox" name="groupBox_movimiento">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>291</width>
         <height>70</height>
        </rect>
       </property>
       <property name="title">
        <string>Mover un eje concreto</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Eje:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxMover1">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMover1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-90</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnMover1">
          <property name="text">
           <string>Mover</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_todos">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>291</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>Mover ejes</string>
       </property>
       <widget class="QSpinBox" name="spinEje0">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnMoverTodos">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Mover</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEje1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEje2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEje3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEje4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinEje5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="interfaz_robot.qrc"/>
 </resources>
 <connections/>
</ui>
